cli/service.go

```diff
@@ -256,7 +257,11 @@ func setupLogger(opts *options.LoggerConfig) error {
 // the flags set in the cli.Context.
 func buildRunCommand(c *cli.Context, serviceType string) []string {
 	args := unparseFlagSet(c, serviceType)
-	subCmd := []string{JasperCommand, ServiceCommand, RunCommand, serviceType}
+	var subCmd []string
+	if filepath.Base(os.Args[0]) != JasperCommand {
+		subCmd = append(subCmd, JasperCommand)
+	}
+	subCmd = append(subCmd, ServiceCommand, RunCommand, serviceType)
```

Comment:
It could happen but I think it's not likely because the Jasper CLI is only used in two places (in this repo and in Curator) and the way CLIs subcommands are structured with `github.com/urfave/cli`, it seems unlikely that this would cause issues. This check for the invoking command's name handles the special case for which this repo's CLI is called "jasper" and does not have "jasper" as a subcommand (because having a CLI that requires you to invoke "jasper jasper" sounds silly, so instead the command it uses is `jasper service run`). For CLIs that are not called jasper (e.g. curator), the Jasper CLI should only be used by invoking the jasper subcommand (e.g. `curator jasper service run`).

Reply:
SGTM